api/routes/apds/activities/goals/openAPI.js

```diff
@@ -28,9 +28,14 @@ const openAPI = {
                   'The description of this goal. Goal is not added if description is missing.'
               },
               objectives: arrayOf({
-                type: 'string',
-                description:
-                  'The description of the objective associated with this goal'
+                type: 'object',
+                properties: {
+                  description: {
+                    type: 'string',
+                    description:
+                      'The description of the objective associated with this goal'
+                  }
+                }
```

Comment:
`synchronize` is generic so it doesn't understand how to turn a single string into a model.  So, don't accept single strings anymore and instead take an object that fully represents the model.  Nobody is using this part of the API so there's nothing to break.  😛

Reply:
👍